pkg/terraform/store.go

```diff
@@ -66,14 +71,22 @@ func WithFs(fs afero.Fs) WorkspaceStoreOption {
 	}
 }
 
+// WithProviderRunner sets the ProviderRunner to be used.
+func WithProviderRunner(pr ProviderRunner) WorkspaceStoreOption {
+	return func(ws *WorkspaceStore) {
+		ws.providerRunner = pr
+	}
+}
+
 // NewWorkspaceStore returns a new WorkspaceStore.
 func NewWorkspaceStore(l logging.Logger, opts ...WorkspaceStoreOption) *WorkspaceStore {
 	ws := &WorkspaceStore{
-		store:    map[types.UID]*Workspace{},
-		logger:   l,
-		mu:       sync.Mutex{},
-		fs:       afero.Afero{Fs: afero.NewOsFs()},
-		executor: exec.New(),
+		store:          map[types.UID]*Workspace{},
+		logger:         l,
+		mu:             sync.Mutex{},
+		fs:             afero.Afero{Fs: afero.NewOsFs()},
+		executor:       exec.New(),
+		providerRunner: NewNoOpProviderRunner(),
```

Comment:
nitpick: since jet-template uses grpc by default, we can also use it by default, too, and document in the guide that you can use NoOp one if shared grpc doesn't work for you for some reason.

Reply:
I would prefer to have an initial set of providers in the wild for some time before making this the default. What do you think?